azure-auth-helper/src/main/java/com/microsoft/azure/auth/WebAppServer.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.auth;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import org.apache.commons.lang3.StringUtils;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.net.InetSocketAddress;
+import java.util.Map;
+
+@SuppressWarnings("restriction")
+public class WebAppServer {
```

Comment:
WebAppSerer is misleading. Use LocalAuthServer instead.

Reply:
Done